.gitignore

```diff
@@ -18,3 +18,4 @@ website/node_modules
 *.bak
 *~
 .*.swp
+.idea/
```

Comment:
I think this is a metadata directory for the IntelliJ IDEA IDE.

@hngkr, rather than checking in a change to each repository to ignore your editor's cache/config files, it's better to configure this sort of thing in your global git environment so it will apply to all repositories you work on. Github has some documentation on this here: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Given that everyone has different editor preferences, it doesn't really scale for each project's ``.gitignore`` to grow to include all of the different sorts of temporary files each editor creates.

Arguably this should've applied to whoever added the ``*.swp`` (Vim, I think) and ``*~`` (Emacs) patterns here, and I'd suggest they ought to be dropped with the same advice as I gave you above. However, it's best not to mix this sort of thing in with other code changes since changes that touch many different parts of the codebase cause a confusing git history and can make merge conflicts more likely, etc.